docs/DeveloperGuide.md

```diff
@@ -426,6 +432,36 @@ An instance of `CommandResult` is then created which contains the message and in
    Applicants matching at least one keyword will be returned (i.e. OR search). 
    e.g. `find Ben Bobby` will return Ben Yang, Bobby Chin.
 
+### Design considerations
```

Comment:
I think this heading can be made to 4 #### so that it doesnt show up on the right side table of contents, I realise it might have come from my mistake in filter, so if you could help me change that heading too that would be great! then the headers for Aspect should be 5 ##### too 😁